code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -295,3 +295,12 @@
 			user << "Nearest operative detected is <i>[nearest_op.real_name].</i>"
 		else
 			user << "No operatives detected within scanning range."
+			
+/obj/item/weapon/pinpointer/wizard
+	name = "wiz pointer"

```

Comment:
That effect is so broken it probably wouldn't make a difference, but no, the vast majority of wizard rounds I don't expect these trackers to come into play, lich still needs that downside in the likely event that trackers are not purchased.